app/tests/src/actions/wallet-actions.js

```diff
@@ -75,28 +75,3 @@ it('Should properly load shelley wallet', async () => {
 
   mockNet.clean()
 })
-
-it('Should properly load byron wallet', async () => {
```

Comment:
this test didn't pass anymore I guess, but that means that corresponding tests for the multi-account feature need to be added to ensure the logic does not get broken over time - it's critical to derive the accounts right, otherwise the users may lose funds

Reply:
Yes, it didn't pass anymore since it was using the byron-wallet that has no notion of accounts so I removed it. And of course, tomorrow I 'll add tests for the multi-account support.